dashboard/app/models/sections/section.rb

```diff
@@ -409,6 +409,21 @@ def assigned_csa?
     script&.csa? || unit_group&.family_name == CSA
   end
 
+  def reset_code_review_groups(new_groups)
+    ActiveRecord::Base.transaction do
+      code_review_groups.destroy_all
+      new_groups.each do |group|
+        # skip any unassigned members
+        next if group[:unassigned]
+        new_group = CodeReviewGroup.create!(name: group[:name], section_id: id)
+        next unless group[:members]
+        group[:members].each do |member|
+          CodeReviewGroupMember.create!(follower_id: member[:follower_id], code_review_group_id: new_group.id)
```

Comment:
Similar question here to our [discussion on Slack](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1635967223178400): are we expecting that the client will convert keys to snake case before invoking this API? I feel like that's a reasonable expectation, but I have seen other places where controller assumes that the data has camel case keys.

Reply:
good question, I don't have a strong preference here and am fine converting to assuming the client will send camel case. I think that's our usual pattern?

Reply:
looking at the sections controller further, it looks like we are generally assuming snake case from the client, so I think I will leave as is.